multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -206,7 +206,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("Manga Rocky", "https://mangarocky.com", "en", overrideVersionCode = 1),
         SingleLang("Manga SY", "https://www.mangasy.com", "en", overrideVersionCode = 1),
         SingleLang("Manga Şehri", "https://mangasehri.com", "tr", className = "MangaSehri", isNsfw = true),
-        SingleLang("Manga Sepeti", "https://www.mangasepeti.com", "tr"),
+        SingleLang("Manga Sepeti", "https://www.mangasepeti.xyz", "tr", overrideVersionCode = 2),
```

Comment:
Use `1`, otherwise it will skip two versions, since it starts with `0`.

```suggestion
        SingleLang("Manga Sepeti", "https://www.mangasepeti.xyz", "tr", overrideVersionCode = 1),
```